js/src/sdk/models/actions.ts

```diff
@@ -276,9 +276,12 @@ export class Actions {
       const { data: res } = await apiClient.actionsV2.advancedUseCaseSearch({
         query: {
           apps: data.apps.join(","),
-          useCase: data.useCase,
+          useCase: "",
```

Comment:
Setting an empty string for `useCase` in the query parameters while it's marked as required in the schema (`required: ["useCase"]`) could lead to validation errors. Consider removing the query parameter entirely since it's being moved to the body.